frontend/coprs_frontend/coprs/templates/_helpers.html

```diff
@@ -157,6 +157,12 @@
     <span class="pficon pficon-ok"></span> {{ state }}
   {% elif state == "failed" %}
     <span class="pficon pficon-error-circle-o"></span> {{ state }}
+    {% if failed_build_log_url and config.LOG_DETECTIVE_BUTTON %}
+      {% set url = "https://log-detective.com/explain?url=%s" %}
+      <a href="{{ failed_build_log_url | format(url) }}" class="btn btn-default btn-xs">
+        <span class="pficon pficon-info"></span> Explain with Log Detective AI
+      </a>
+    {% endif %}
```

Comment:
Can we place this into a separate column, since this is not a log file but "request for explanation".

Reply:
Or, maybe we could have a clickable "failed" state? Can you upload a screenshot?